### ISSUE TITLE
Predefined zoom level, toggle off zoom in/out buttons | Android Version

### ISSUE THREAD

**Issue by ah-dagdelen** [bk138/multivnc#236]
Hi,

On a 10.5" tablet., I've been using Android version of MultiVNC downloaded from F-Droid.
MultiVNC version: 2.1.4 - Android version: 12.

I would like to suggest two features related to this version:

1) To save some screen estate, it would be useful to toggle off zoom in/out and keyboard buttons, which are located on the bottom side of the screen. After connecting a bluettooth keyboard and setting the zoom level, they unnecessarily occupy screen estate.

2) Predefined and saved zoom level, like 150%, before connecting to Server would be also useful. In this version, one has to manually set zoom level after every connection. It would be nice if MultiVNC could connect and show the screen with predefined zoom level.

Thanks a lot for the app!

**Comment by elheremes** [comment]
Agree with this, would be great if at least I can hide UI buttons when connected.

**Comment by unique-EJ** [comment]
Yes, to set a predefined zoom level in bookmarked (saved) servers would be a feature that helps to use the app.
